kitsune/search/v2/search.py

```diff
@@ -203,6 +239,23 @@ def get_filter(self, **kwargs):
             "bool", should=self._from_children("get_filter", **kwargs), minimum_should_match=1
         )
 
+    def get_highlight_options(self):
+        return {
+            "fields": self.get_highlight_fields(),
+            "type": "fvh",
+            # order highlighted fragments by their relevance:
+            "order": "score",
+            # only get one fragment per field:
+            "number_of_fragments": 1,
+            # split fragments at the end of sentences:
+            "boundary_scanner": "sentence",
+            # return fragments roughly this size:
+            "fragment_size": SNIPPET_LENGTH,
+            # add these tags before/after the highlighted sections:
+            "pre_tags": [f"<{HIGHLIGHT_TAG}>"],
+            "post_tags": [f"</{HIGHLIGHT_TAG}>"],
+        }
+
```

Comment:
This means that bare `QuestionSearch` or `WikiSearch` don't get any highlighting. Can we just conditionally add this in `SumoSearch` if `len(self.get_highlight_fields) > 0`?